src/Global.scss

```diff
@@ -417,3 +417,51 @@ textarea {
 .mw-1200{
   max-width: 1200px;
 }
+
+@keyframes shimmer {
+  0% {
+      background-position: -1000px 0;
+  }
+  100% {
+      background-position: 1000px 0;
+  }
+}
+
+.shimmer-custom {
+  background: linear-gradient(to right, #617789 8%, #3c4b57 18%, #617789 33%);
+  background-size: 1000px 100%;
+  animation: shimmer 2.2s linear infinite forwards;
+}
+
+.rect {
```

Comment:
classes like `.rect` are way to global and are likely to mess with other things
```CSS
...
.shimmer-custom {
  background: linear-gradient(to right, #617789 8%, #3c4b57 18%, #617789 33%);
  background-size: 1000px 100%;
  animation: shimmer 2.2s linear infinite forwards;
  .rect {
    width: 100%;
    aspect-ratio: 100 / 86;
    margin-bottom: 20px;
  }
}
...
```
Something like this would do.
Assuming that the HTML is structured in such a way that .rect is in fact a child of .shimmer-custom